.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -0,0 +1,47 @@
+name: Scanner dev vulnerability update
+on:
+  schedule:
+  - cron: "30 */4 * * *"
```

Comment:
Every 4 hours seems reasonable and do it on the 30 minute mark just to offset it from other jobs which probably run on the 0 minute mark